plugins/geospatial/server/geospatial.py

```diff
@@ -263,15 +263,15 @@ def near(self, params):
                         raise ValueError
 
                 except ValueError:
-                    raise RestException("Parameter '{}' must be a number."
-                                        .format(param))
+                    raise RestException("Parameter '%s' must be a number." %
+                                        param)
 
-                condition['$'+param] = distance
+                condition['$%s' % param] = distance
```

Comment:
`+` actually works fine in cases like this, it coerces unicode gracefully with the same behavior as `%`.